### DISCUSSION TITLE
server build of solid-js/web doesn't export SVGElements

### DISCUSSION THREAD

**Discussion by JLarky** [solidjs/solid#855] [General]
```
Watcher File change detected! Restarting!
Check .../main.tsx
error: Uncaught SyntaxError: The requested module 'solid-js/web' does not provide an export named 'SVGElements'
import { spread, assign, insert, createComponent, dynamicProperty, SVGElements } from 'solid-js/web';
                                                                   ~~~~~~~~~~~
    at <anonymous> (https://unpkg.com/solid-js@1.3.7/h/dist/h.js:1:68)
Watcher Process finished. Restarting on file change...
```

you can see that https://unpkg.com/solid-js@1.3.7/h/dist/h.js is importing `SVGElements` from `solid-js/web` but `SVGElements` is not defined in server build, only for client version of `soild-js/web`

**Comment by arstnei0** [comment]
Is this problem solved?

**Comment by millette** [comment]
See also #1480 and #1483.

**Comment by ryansolid** [comment]
I have been trying to figure out what expectation is here. solid-js/h and server build are not compatible. `h` actually produces DOM nodes. I am not even certain I can get a string-rendered compatible version of h working, along with a hydration-capable version as well. It's a ton of work and it might not pan out.

Basically fixing this error will just lead you to the next one.

If you want to use Solid JSX on the server use our transform.
